### PR TITLE
Fix truncated html log

### PR DESCRIPTION
**Pull request by arcusfelis**
This PR addresses "MIM-2348 Logs are truncated on CI in the links. Increase max_no_bytes to 10 megabytes"

Proposed changes include:
* Increase size of maximum log file size from 2MB to 10MB.
* Disable detailed logs for global distribution - this would reduce amount of logs. 
* And still there is too much noise in logs from GD trying to stop connections - changed the loglevel, if connections are stopped because of shutdown.
* Enable logs for mim2/mim3 - but without printing INIT/DONE for tests (that is configurable). Reason is to debug clustering logic, which is often fails on CI nowadays.

We used to disable html logs for mim2/mim3 because it breaks `--one-node` flag in `test-runner`. But now we check if the node is running, and if not - the hook would not do anything.

Example of INIT/DONE:
```
INIT suite=mod_http_upload_SUITE group=mod_http_upload_s3 testcase=get_url_ends_with_filename
DONE suite=mod_http_upload_SUITE group=mod_http_upload_s3 testcase=get_url_ends_with_filename
```

### CODE REVIEW

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_27](https://circleci.com/gh/esl/MongooseIM/246365) / elasticsearch_and_cassandra_mnesia / 96ed7a4a54a5b27300ace8ef75615808003d9aed
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4416/246365/elasticsearch_and_cassandra_mnesia.27.1.2-amd64)/ [big](https://esl.github.io/html-zip-reader/PR/4416/246365/elasticsearch_and_cassandra_mnesia.27.1.2-amd64/big.tar.gz//ct_run.test%4009f38eaa1308.2024-12-04_15.21.06/index.html)
**OK: 473** / Failed: 0 / **User-skipped: 49** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_27](https://circleci.com/gh/esl/MongooseIM/246380) / elasticsearch_and_cassandra_mnesia / 77042ea8977d2cb62258b7bf92ab4d4faa1acd72
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4416/246380/elasticsearch_and_cassandra_mnesia.27.1.2-amd64)/ [big](https://esl.github.io/html-zip-reader/PR/4416/246380/elasticsearch_and_cassandra_mnesia.27.1.2-amd64/big.tar.gz//ct_run.test%4056726b698bae.2024-12-04_15.28.59/index.html)
**OK: 473** / Failed: 0 / **User-skipped: 49** / Auto-skipped: 0

File: big_tests/src/ct_mongoose_log_hook.erl [hunk 1]
```diff
@@ -288,9 +310,13 @@ connect_mim_node(HookOpts) ->
...
     ct:pal("connecting to the '~p' node (cookie: '~p')", [Node, Cookie]),
-    %% crash if cannot connect to the node
-    true = net_kernel:connect_node(Node),
-    Node.
+    case net_kernel:connect_node(Node) of
+        true ->
+            Node;
+        _ ->
+            %% Could happen if we test not with all nodes enabled.
+            false
```
Comment (NelsonVides) [inline big_tests/src/ct_mongoose_log_hook.erl:318]: I think `undefined` is a better meaning name than `false` maybe? 🤔

**Review by NelsonVides — COMMENTED** [review]
Looks very very nice, just a couple of comments 👌🏽

File: big_tests/dynamic_domains.spec [hunk 2]
```diff
@@ -174,7 +174,10 @@
 %% * ensure preset & mim_data_dir values are passed to ct Config
 %% * check server's purity after SUITE
 {ct_hooks, [ct_groups_summary_hook, ct_tty_hook, ct_mongoose_hook, ct_progress_hook,
-            ct_markdown_errors_hook, ct_mongoose_log_hook]}.
+            ct_markdown_errors_hook,
+            ct_mongoose_log_hook,
+            {ct_mongoose_log_hook, [{host, mim2}, {log, [suite, group]}, {print_init_and_done_for_testcases, false}]},
+            {ct_mongoose_log_hook, [{host, mim3}, {log, [suite, group]}, {print_init_and_done_for_testcases, false}]}]}.
```
[the remark this answers is not in the document]
Comment (arcusfelis) [reply big_tests/dynamic_domains.spec:180]: I can remove it at all, just copied from docs randomly. default `log, [suite]` should be fine

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_27](https://circleci.com/gh/esl/MongooseIM/246705) / elasticsearch_and_cassandra_mnesia / 2d33f83d6164b6f1cfa9ccb6b501d18f1f2da5bc
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4416/246705/elasticsearch_and_cassandra_mnesia.27.1.2-amd64)/ [big](https://esl.github.io/html-zip-reader/PR/4416/246705/elasticsearch_and_cassandra_mnesia.27.1.2-amd64/big.tar.gz//ct_run.test%40440263ab117d.2024-12-05_09.14.52/index.html)
**OK: 473** / Failed: 0 / **User-skipped: 49** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_27](https://circleci.com/gh/esl/MongooseIM/246796) / elasticsearch_and_cassandra_mnesia / 97088ea1abc8df0197584c5b2b17ae185d5b998c
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4416/246796/elasticsearch_and_cassandra_mnesia.27.1.2-amd64)/ [big](https://esl.github.io/html-zip-reader/PR/4416/246796/elasticsearch_and_cassandra_mnesia.27.1.2-amd64/big.tar.gz//ct_run.test%4097bf06cbd6f3.2024-12-05_11.14.36/index.html)
**OK: 473** / Failed: 0 / **User-skipped: 49** / Auto-skipped: 0

**Review by NelsonVides — APPROVED** [review]
LGTM 👌🏽

**Pull request merged**